### PR TITLE
Issue #991 fixed

### PR DESCRIPTION
**Pull request by XFilow**
Qt: Encryption dialog bug fixed.

Changes include:
Showing a dialog message while encrypting wallet for user acknowledgment.
Closing the askpassphrase dialog before encryption starts instead of after in order for it not to bug.

### CODE REVIEW

**Comment by schinzelh** [comment]
Shouldn't these files be located in `src/qt`

**Comment by XFilow** [comment]
Absolutely, my bad... I moved the files to the wrong folder.

**Comment by XFilow** [comment]
This second commit was not intended, was just trying to sync my repo, sorry!

File: src/qt/walletmodel.cpp [hunk 1]
```diff
@@ -36,17 +36,9 @@ WalletModel::WalletModel(const PlatformStyle *platformStyle, CWallet *wallet, Op
...
     transactionTableModel(0),
     recentRequestsTableModel(0),
-    cachedBalance(0),
-    cachedUnconfirmedBalance(0),
-    cachedImmatureBalance(0),
-    cachedAnonymizedBalance(0),
-    cachedWatchOnlyBalance(0),
-    cachedWatchUnconfBalance(0),
-    cachedWatchImmatureBalance(0),
+    cachedBalance(0), cachedUnconfirmedBalance(0), cachedImmatureBalance(0),
```
Comment (tgflynn) [inline src/qt/walletmodel.cpp]: It looks like you've picked up an old version of this file.  You're patch is reverting some bug fixes that were made recently.

**Comment by UdjinM6** [comment]
For me (on Mac) it only shows a notification once which then hides in a side bar, which does not make much of a difference. Would be better to use a small dialog like the one on shutdown or rescan imo.

**Comment by nmarley** [comment]
This is ancient at this point. I believe we can close this and come back to the PR in the future if necessary.

**Pull request closed without merging**